tests/plugins/easyimage/_helpers/tools.js

```diff
@@ -6,35 +6,7 @@ var easyImageTools = ( function() {
 
 	// Force Edge to run every test in new CKEditor's instance.
 	function createTestsForEditors( editors, tests ) {
```

Comment:
This function is now used only in one test in easyimage - `commands.js`, so you may just remove this helper (as it acts as very simpler proxy now) and add `bender.tools.createTestsForEditors( editors, tests, CKEDITOR.env.edge );` call directly in that test.